src/cpr_sdk/cli/search.py

```diff
@@ -141,10 +142,10 @@ def main(
             family_url = f"https://app.climatepolicyradar.org/document/{family_data['family_slug']}"
             details = f"""
             [bold]Total hits:[/bold] {len(family.hits)}
-            [bold]Family:[/bold] [link={family_url}]{family_data['family_import_id']}[/link]
-            [bold]Family slug:[/bold] {family_data['family_slug']}
-            [bold]Geography:[/bold] {family_data['family_geography']}
-            [bold]Relevance:[/bold] {family_data['relevance']}
+            [bold]Family:[/bold] [link={family_url}]{family_data["family_import_id"]}[/link]
+            [bold]Family slug:[/bold] {family_data["family_slug"]}
+            [bold]Geography:[/bold] {family_data["family_geography"]}
+            [bold]Relevance:[/bold] {family_data["relevance"]}
```

Comment:
noticed this kind of change in another place too. I wonder whether either your local formatting config is different from CI, or CI is broken?

Reply:
yeah, i noticed this too and i'm not sure what the cause is... pre-commit etc are all installed locally and things seem to be passing on both this branch and main, so i guess i might have some extra local rules which this repo's formatter doesn't care about? 🤷 doesn't seem like something which needs to be urgently fix imo, but i'll poke around over the next day or two while this work continues to develop

Reply:
no rush and obv not important! had just seen it and thought i'd call it out in case it ended up causing you surprise headaches down the road